map_data/definition.csv

```diff
@@ -603,7 +603,7 @@
 602;25;163;133;SOUTH SEA;x;
```

Comment:
You need to move terrain maps, etc. from `Warcraft-Guardians-of-Azeroth-2\gfx\map\terrain` and `watercolor_rgb_waterspec_a.dds` from `Warcraft-Guardians-of-Azeroth-2\gfx\map\water`.

Also, tree masks from `Warcraft-Guardians-of-Azeroth-2\content_source\map_objects\masks`.